src/server/constants.py

```diff
@@ -13,3 +13,17 @@ class PATHS:
     DATABASE_SCHEMA = RESOURCES + "schema.sql"
     DATABASE_DEFAULT_DUMP = RESOURCES + "dump.sql"
     DATABASE = VAR_PATH + "database_ctb.db"
+
+
+class QUERIES:
```

Comment:
Please add @dataclass(frozen=True) above `class Queries:` so that fields of this class become immutable